.github/workflows/ci.yaml

```diff
@@ -17,38 +17,22 @@ jobs:
     - name: Free space
       run: |
         sudo rm -rf /usr/local/android /usr/share/dotnet /usr/local/share/boost /opt/ghc /usr/local/share/chrom* /usr/share/swift /usr/local/julia* /usr/local/lib/android /usr/local/graalvm /usr/local/aws* /usr/local/lib/heroku 
-        du . --max-depth=1 -h
-        ls -la
-        cd /usr/local
-        du . --max-depth=1 -h
-        ls -la
-        cd /usr/local/lib
-        echo $(pwd)
-        du . --max-depth=1 -h
-        ls -la
 
     - uses: actions/checkout@v3
 
     # In this step, this action saves a list of existing images,
     # the cache is created without them in the post run.
     # It also restores the cache if it exists.
-    # name: Docker Layer Caching2
     - uses: jpribyl/action-docker-layer-caching@v0.1.1
       # Ignore the failure of a step and avoid terminating the job.
       continue-on-error: true
 
-    - name: Prepare timestamp
-      id: cache_timestamp
-      shell: bash 
-      run: echo timestamp="$(date +'%Y-%m-%dT%H:%M:%S')" >> $GITHUB_OUTPUT
-
     - name: Cache files for tidy
-      uses: pat-s/always-upload-cache@v3.0.11
+      uses: actions/cache@v3 
       with:
         path: tidy-cache
-        key: tidy-cache-${{ steps.cache_timestamp.outputs.timestamp }}
+        key: tidy-cache-${{ github.ref }}
```

Comment:
This should be `github.run_id` otherwise the cache will never update on the branch.

Reply:
That would be similar to using timestamp. It would generate different each time for each run and would miss and save cache each time.